testing/tidyverse_functions.R

```diff
@@ -0,0 +1,230 @@
+library(magrittr)
+#library(dplyr)
+#library(stringr)
+
+setwd("D:/OneDrive/University of Auckland/Summer Research Scholarship/iNZight work")
+
+stats20x.df <- read.csv("stats20x.csv", header = TRUE)
+
+# DATASET -> FILTER DATASET -> LEVELS OF A CATEGORICAL VARIABLE
+###---------------------------------------------------------
+
+# example : filtering for column "DEGREE" for only obs with "BCom", "BA" or "Other"
+filtered.DEGREE <- stats20x.df %>% 
+  dplyr::filter(DEGREE %in% c("BCom", "BA", "Other")) %>% 
+    droplevels()
+###
+
+
+# nse function
+datasetFilterCat <- function(dataset, var, levels){
```

Comment:
Let's place these functions at the end of the file, and keep the raw tidyverse code all together to make it easier to show others.